js/models/roomcollection.js

```diff
@@ -28,7 +28,9 @@
 
 	var RoomCollection = Backbone.Collection.extend({
 		model: OCA.SpreedMe.Models.Room,
-		comparator: 'name',
+		comparator: function(model) {
+			return -(model.get('lastPing'));
```

Comment:
When we 'ping' the current room, a timestamp is set in 'lastPing'
https://github.com/nextcloud/spreed/blob/master/lib/Controller/ApiController.php#L280